test/bigtest/tests/loan-policy/loan-policy-detail-test.js

```diff
@@ -25,6 +25,40 @@ describe('LoanPolicyDetail', () => {
   let loanPolicy;
 
   describe('viewing loan policy', () => {
+    describe('\n\taccordions\n', () => {
+      beforeEach(function () {
+        loanPolicy = this.server.create('loanPolicy', {
+          loanable: true,
+        });
+
+        this.visit(`/settings/circulation/loan-policies/${loanPolicy.id}`);
+      });
+
+      it('should be displayed', () => {
+        expect(LoanPolicyDetail.expandAll.isPresent).to.be.true;
+      });
+
+      describe('\n\tcollapse all\n', () => {
```

Comment:
Just for curiosity, why do you use `\n`?

Reply:
In case of testing accordions, it's useless(thank you for pointing this out). But I'm using such control characters to separate sections in tests.
![Screen Shot 2019-03-12 at 11 12 32 AM](https://user-images.githubusercontent.com/42577309/54189055-acc8f380-44b9-11e9-913a-34055494f4ec.png)

Reply:
Sounds good.